core/documents-manager.php

```diff
@@ -257,11 +300,18 @@ public function get_doc_or_auto_save( $id, $user_id = 0 ) {
 	 * @return false|Document The document if it exist, False otherwise.
 	 */
 	public function get_doc_for_frontend( $post_id ) {
-		if ( is_preview() || Plugin::$instance->preview->is_preview_mode() ) {
-			$document = $this->get_doc_or_auto_save( $post_id, get_current_user_id() );
-		} else {
-			$document = $this->get( $post_id );
-		}
+		// if ( is_preview() || Plugin::$instance->preview->is_preview_mode() ) {
+		// 	$document = $this->get_doc_or_auto_save( $post_id, get_current_user_id() );
+		// } else {
+		// 	$document = $this->get( $post_id );
+		// }
+
+		// echo 'mykit'.$post_id.'<br />';
+		// echo get_the_title( $post_id );
+
+		$document = $this->get( $post_id );
+
+		// var_dump( $document );
```

Comment:
Same here @hein-obox Please talk to Louis, make sure there's no mixed up code.

Reply:
* Sorry, I accidentely merged some other files while working late *